src/main/java/com/playonlinux/wine/WinePrefix.java

```diff
@@ -41,6 +44,7 @@
     private static final String EXECUTABLE_EXTENSION = "exe";
     private static final String DRIVE_C = "drive_c";
 
+    private static String[] EXCLUDES_FILES = new String[1]; // One element for now
```

Comment:
It is better to directly set here the list of exclusion:
private static String[] EXCLUDES_FILES = new String[] { "iexplore.exe", "notepad.exe" }, etc...